docs/learn/learn-polkadot-opengov.md

```diff
@@ -575,6 +575,15 @@ stakeholder voting for a proposed referendum. Members of the Fellowship can vote
 Fellowship proposal and the aggregated opinion of the members (weighted by their rank) constitutes
```

Comment:
at some point we should probably have a good reference for how ranks exactly work